styles.css

```
@@ -13,3 +13,320 @@
 }
 
 /* 🦊 Vos styles ici */
+body {
+  background-color: #ebe7d6;
+  margin: 0px;
+  font-family: "Fredoka", sans-serif;
+
+
+
+}
+
+h1 {
+  color: #5a4132;
+
+}
+
+p {
+  color: #5a4132;
+}
+
+h2 {
+  color: #5a4132;
+  font-weight: 450;
+}
+
+h3 {
+  color: #5a4132;
+  font-weight: 350;
+  font-size: 15px;
+  margin-top: -90px;
+  margin-left: 20px;
+
+}
+
+.content {
+  width: 430px;
+  height: 1900px;
+  background-color: #fefcf1;
+  border-radius: 25px 25px 0px 0px;
+  margin-top: 300px;
+
+
+}
+
+.livre {
+  text-align: center;
+  margin-top: 30px;
+
+}
+
+.etoile {
+  width: 150px;
+  display: flex;
+  margin-left: 17px;
+  margin-top: -30px;
+
+}
+
+
+
+
+.bloc-livre {
+  transform: translateY(-290px);
+}
+
+.plus {
+  text-align: end;
+}
+
+h1 {
+  text-align: center;
+}
+
+h2 {
+  text-align: center;
+}
```

Si les 2 nécessitent cette propriété, on peut simplifier en:

```suggestion
h1, h2 {
  text-align: center;
}
```

Ces 2 éléments HTML sont d'ailleurs déjà stylés aux lignes 34 et 39.